release-notes/major12/12.0.0/migration.md

```diff
@@ -74,6 +74,19 @@ This page describes how to update Valtimo from the previous version to the curre
 
        If the `value` field contains an actual email, then the email must be changed to a user ID.
 
+* **Value resolvers**
+
+  Scope: back-end
+
+    1. **Step1**
```

Comment:
These are placeholder step titles. Can you replace them with titles that make sense? I think steps 1 and 2 can be combined, by the way.

Reply:
Changed